sos/report/plugins/ceph_common.py

```diff
@@ -41,45 +43,52 @@ class Ceph_Common(Plugin, RedHatPlugin, UbuntuPlugin):
 
     # This check will enable the plugin regardless of being
     # containerized or not
-    files = ('/etc/ceph/ceph.conf',)
+    files = ('/etc/ceph/ceph.conf',
+             '/var/snap/microceph/*',)
 
     def setup(self):
         all_logs = self.get_option("all_logs")
 
-        self.add_file_tags({
-            '.*/ceph.conf': 'ceph_conf',
-            '/var/log/ceph(.*)?/ceph.log.*': 'ceph_log',
-        })
-
-        if not all_logs:
-            self.add_copy_spec("/var/log/calamari/*.log",)
+        if not os.path.exists('/var/snap/microceph/common'):
```

Comment:
Use `self.path_exists()` here which is built into `Plugin` so that we can automatically account for when sos is run in a container or with a non-/ sysroot.

Reply:
Thank you, I will make those changes